datastore.py

```diff
@@ -0,0 +1,38 @@
+import boto3
+import json
+
+from botocore.errorfactory import ClientError
+
+s3 = boto3.resource('s3')

```

Comment:
What's the reason for using `resource` here instead of `client`? What's the difference? Would client not work here? Does it matter?

Reply:
it’s just preference. they can both be used.

Reply:
the difference is just syntax. like
```python
s3 = boto3.client(‘s3’)
s3.put_object(bucket, key, value)
```

versus
```python
s3 = boto3.resource(‘s3’)
bucket = s3.bucket(‘xyz’)
obj = bucket.key(‘abc’)
obj.put(val)
```